package/ci/circleci.yml

```diff
@@ -315,7 +315,7 @@ jobs:
       CONFIGURATION: Debug
     steps:
     - install-base-linux:
-        extra: libfaad-dev libfreetype6-dev libjpeg-dev libopenal-dev libpng12-dev libdevil-dev libharfbuzz-dev libassimp-dev wget
+        extra: libfaad-dev libfreetype6-dev libjpeg-dev libopenal-dev libpng12-dev libdevil-dev libharfbuzz-dev libassimp-dev wget libwebp-dev
```

Comment:
@mosra Is it possible that additions here require approval/whitelisting from you? I believe @melikebatihan is getting configuration errors on CircleCI, forwarding:

![image](https://user-images.githubusercontent.com/1889284/169662030-9b36f9ac-a50c-4f96-8b06-65f8a080babe.png)